src/Entity/ServerManager/GameList.php

```diff
@@ -205,6 +205,10 @@ public function setGameEndMonth(int $gameEndMonth): self
 
     public function getGameCurrentMonth(): int
     {
+        // taken from api/v1/Game.php GetCurrentMonthAsId()
+        if ($this->gameState == 'SETUP') {
+            $this->gameCurrentMonth = -1;
+        }
```

Comment:
please sift this out for a separate merge into dev so it can become part of 4.0.2